tests/comprehension_dict.py.uast

```diff
@@ -1,5 +1,6 @@
-Status:  ok
+Status:  error
```

Comment:
Is this status change expected? The same question applies to the next UAST files